src_c/doc/pygame_doc.h

```diff
@@ -11,6 +11,7 @@
 #define DOC_PYGAMEREGISTERQUIT "register_quit(callable) -> None\nregister a function to be called when pygame quits"
 #define DOC_PYGAMEENCODESTRING "encode_string([obj [, encoding [, errors [, etype]]]]) -> bytes or None\nEncode a Unicode or bytes object"
 #define DOC_PYGAMEENCODEFILEPATH "encode_file_path([obj [, etype]]) -> bytes or None\nEncode a Unicode or bytes object as a file system path"
+#define DOC_PYGAMEDEBUG "debug(filename=None) -> None\nretrieves useful information for debugging and issue-reporting purposes"
```

Comment:
again the name of the function looks to be the old name here.

Reply:
Just pushed a commit to fix this